DQMOffline/Configuration/python/DQMOfflineHeavyIons_cff.py

```diff
@@ -30,16 +30,6 @@
                                        DQMMessageLogger )
 
 from DQMOffline.Muon.muonMonitors_cff import *
-diMuonHistos.etaBin = cms.int32(70) #dimuonhistograms mass, bin
-diMuonHistos.etaBBin = cms.int32(70)
-diMuonHistos.etaEBin = cms.int32(70)
-diMuonHistos.etaBinLM = cms.int32(12)
-diMuonHistos.etaBBinLM = cms.int32(12)
-diMuonHistos.etaEBinLM = cms.int32(12)
-diMuonHistos.LowMassMin = cms.double(2.0)
-diMuonHistos.LowMassMax = cms.double(14.0)
-diMuonHistos.HighMassMin = cms.double(55.0)
-diMuonHistos.HighMassMax = cms.double(125.0)
```

Comment:
the nominal policy is for everything aiming for 80X to first have an equivalent in 81X